src/F2_autoreport.py

```diff
@@ -0,0 +1,119 @@
+import dash
+import dash_html_components as html
+import time
+
+app = dash.Dash(__name__)
+
+colors = {
+    'bg-head': '#9ae6db',
+    'text-head': '#000000',
+    'text-body': '#000000',
+    'inp-box': '#03034f',
+    'font-inpbox': '#FFFFFF'
+}
+
+projectName = 'Harbor Norway'
+scenarioName = '100% self-generation'
+releaseDesign = '0.0x'
+branchID = 'xcdd5eg004'
+simDate = time.strftime("%Y-%m-%d")
+
+# Header section with logo and the title of the report, and CSS styling. Work in progress...
+
+app.layout = html.Div([
+
+    html.Div(className='header_title_logo',
+             children=[html.Img(id='mvslogo', src='https://elandh2020.eu/wp-content/themes/netron/libs/images/logo'
+                                                  '/logo-eland-original.svg', width='370px'),
+                       html.H1('MULTI VECTOR SIMULATION - REPORT SHEET')],
+             style={
+                 'textAlign': 'center',
```

Comment:
It is better to save all the style setting in a class def in a cssfile, then use the prop `className` to link to the class. For small project inline styling is ok though.

Reply:
You can save the css file in a folde rnamed `assets` then dash will automatically load it (the folder should be in the same path as the file defining the app)